src/NServiceBus.AcceptanceTests/Core/Recoverability/When_delayed_and_immediate_retries_with_serialization_exception.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Recoverability
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using MessageMutator;
+    using NUnit.Framework;
+
+    public class When_delayed_and_immediate_retries_with_serialization_exception : NServiceBusAcceptanceTest
```

Comment:
hm, title doesn't seem to really align any more?
How about `When_failing_mutated_message`?